src/CountryList.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Umpirsky;
+
+abstract class CountryList
+{
+
+    static function get(string $language): array
+    {
+	$path = __DIR__.'/../data/'.$language;
+
+	if(!is_dir($path)) {
+	    throw new \Exception("Language $language not found.");
+	}
+
+	return require $path.'/country.php';
```

Comment:
We should check if `$path.'/country.php'` exists too.

Reply:
Do you want to throw nice exception? `require` will fail anyway.

Reply:
Exception is better.

Reply:
Why are you not merging this?

Reply:
https://github.com/umpirsky/country-list/pull/99#issuecomment-575591400